warehouse/core/src/main/java/datawave/ingest/util/cache/Loader.java

```diff
@@ -165,6 +165,10 @@ public void run() {
         
     }
     
+    public boolean equals(Loader loader) {
+        return (this.hashCode() == loader.hashCode());
+    }
+    
     @Override
     public int hashCode() {
         return System.identityHashCode(this) + childHash;
```

Comment:
I know you didn't add this, but why not create a legit hashcode, like this:
```suggestion
        return Objects.hash(entryCache, executor, children, childHash, lazy);
```

Reply:
:+1: